src/main/java/Duke.java

```diff
@@ -1,10 +1,127 @@
+import java.util.Scanner;
+
 public class Duke {
+    public static Task[] list = new Task[100];
+    public static int taskLength = 0;
+
+    //add a new task
+    public static void handleTasks(String input) {
+        int index = input.indexOf(" ");
+        String taskType = input.substring(0, index);
+        String taskContent = input.substring(index);
+        String taskFinal = taskContent;
+
+        switch (taskType) {
+        case "deadline":
+            try {
+                String deadlineContent = taskContent.split("/by ")[1];
+                taskFinal = taskContent.split("/")[0] + " (by:" + deadlineContent + ")";
+            }
+            catch (Exception e) {
+                System.out.println("Cannot parse deadline start date! Requires [/by]");
+            }
```

Comment:
According to the Coding Standard, the try-catch statement should be of the following form:
```java
try {
    statements;
} catch (Exception exception) {
    statements;
}